crates/bitwarden/src/commands/login.rs

```diff
@@ -234,7 +238,7 @@ pub(crate) async fn renew_token(client: &mut Client) -> Result<()> {
                 client_secret,
                 ..
             } => {
-                AccessTokenRequest::new(&service_account_id, &client_secret)
+                AccessTokenRequest::new(*service_account_id, &client_secret)
```

Comment:
Haven't seen `*` used before, how does that work?

Reply:
It's a dereference operator, the reverse of &.

We have a &Uuid and want to make it an Uuid, because Uuid is Copy, we can just use * which will copy the value. If it wasn't copy we'd be using clone() or to_owned() instead